sdks/python/apache_beam/utils/timestamp.py

```diff
@@ -66,6 +66,7 @@ def __init__(self, seconds=0, micros=0):
       raise TypeError(
           'Cannot interpret %s %s as micros.' % (micros, type(micros)))
     self.micros = int(seconds * 1000000) + int(micros)
+    self.seconds = seconds + micros / 1000000
```

Comment:
I do not believe we want to add another field to keep track of the same information. A few options:
- In periodicsequence.py read micros and convert to seconds there.
- hava an attribute on Timestamp here to return seconds from the internally stored micros.

(same for the Duration change below)

Reply:
Good point, I was considering using the `@property` and a `_get_seconds()`. What do you think?

Reply:
I have coded both options and I think the first approach looks better and more compact. I will push the version that changes to seconds directly in `periodicsequence`